getting-started/vs-integration/new-project-wizard.md

```diff
@@ -50,31 +52,28 @@ To start the wizard, use either of the following options:
 
 ## Using the Wizard
 
-In the order of their appearance, the Create New Project wizard presents the following screens with options that will assist you to configure your new app:
-
-1. On the first screen, you can select a location and a name for your project.
+In the order of their appearance, the Create New Project wizard presents several screens with options that will assist you to configure your new app.
```

Comment:
This is the last instance of bold formatting for the wizard name that I am suggesting. Please, apply as needed throughout.

```suggestion
In the order of their appearance, the **Create New Project** wizard presents several screens with options that will assist you to configure your new app.
```